locales/en-US/app.ftl

```diff
@@ -705,9 +705,12 @@ ServiceWorkerManager--hide-notice-button =
 ## This is the settings component that is used in Call Tree, Flame Graph and Stack
 ## Chart panels. It's used to switch between different views of the stack.
 
-StackSettings--implementation-all-stacks = All stacks
-StackSettings--implementation-javascript = JavaScript
-StackSettings--implementation-native = Native
+StackSettings--implementation-all-stacks2 = All frames
+    .title = Do not filter the stack frames
```

Comment:
Note: I changed "All stacks" to "All frames", tell me what you think

Reply:
Looks good to me.

While we are changing the id, how about making it `StackSettings--implementation-all-frames` instead of `stacks2`?

Reply:
good point!